src/organisms/cards/FeaturedPolicyCard/__tests__/__snapshots__/featured_policy_card.spec.js.snap

```diff
@@ -59,7 +59,7 @@ exports[`<FeaturedPolicyCard /> renders correctly 1`] = `
               style={undefined}
             >
               <p
-                className="text typography-7 regular undefined undefined p undefined"
+                className="text typography-7 regular undefined undefined p undefined policy-info-label"
```

Comment:
Are these `undefined` classes supposed to be here?

Reply:
They aren't. This happens when there isn't a guard clause preventing the class names from being added to the class names array.

Reply:
Gotcha. Thanks for the pro tip! 

Just an FYI @jmcolella, there are a whole lot more in this file than just this  one line.

Reply:
@danielnovograd yeah I see. This looks like a bunch of components would have to be refactored to prevent this. I'll make a note of it for rebrand and see if they can pick it up